guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-by-using-script-rex.adoc

```diff
@@ -0,0 +1,37 @@
+[id="deploying-the-ca-certificate-on-a-host-by-using-script-rex"]
+= Deploying the CA certificate on a host by using Script REX
+
+You can use remote execution (REX) with the Script provider to deploy the CA certificate.
+Redeploy the CA certificate when you change it on {ProjectServer}.
+
+.Prerequisites
+* The host is registered to {Project}.
+* Remote execution is enabled on the host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs*.
+. Click *Run Job*.
+. From the *Job category* list, select `Commands`.
+. From the *Job template* list, select `Download and run a script`.
+. Click *Next*.
+. Select hosts on which you want to execute the job.
+. In the *url* field, enter the following URL:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+https://_{foreman-example-com}_/unattended/public/foreman_ca_refresh
```

Comment:
@ShimShtein This template will refresh the CA cert if and only if Katello is present, right?

Reply:
In that case I exclude the REX variants for non-Katello builds.